src/tests/suite/daos_test_common.c

```diff
@@ -899,8 +900,14 @@ daos_kill_server(test_arg_t *arg, const uuid_t pool_uuid,
 		      "disabled, svc->rl_nr %d)!\n", rank, arg->srv_ntgts,
 		       arg->srv_disabled_ntgts - 1, svc->rl_nr);
 
-	rc = daos_mgmt_svc_rip(grp, rank, true, NULL);
-	assert_int_equal(rc, 0);
+	/* build and invoke dmg cmd to stop the server */
+	dts_create_config(dmg_cmd, "dmg system stop -i --ranks=%d --force",
+			  rank);
+	rc = system(dmg_cmd);
+	if (rc == -1 || WEXITSTATUS(rc) != 0) {
+		print_message("%s failed rc=%d\n", dmg_cmd, rc);
```

Comment:
should this not be an assert (or at least return an error to the caller)?   The prio code asserted